anvil/src/main/kotlin/dev/inkremental/KAnvil.kt

```diff
@@ -25,11 +25,18 @@ inline fun <reified T: View> v(noinline r: () -> Unit = {}) = v(T::class, r)
 
 inline fun <reified T: View, reified S: ViewScope> v(s: S, noinline r: S.() -> Unit = {}) = v(T::class, r.bind(s))
 
+inline fun <reified S: ViewScope>xml(@LayoutRes layoutId: Int, s: S, noinline r: S.() -> Unit = {}) {
+    start(layoutId)
+    r.bind(s).invoke()
+    end()
+}
+
```

Comment:
For start() and end() agree, let's hide them. But for hide() I would not hide it, at least for now, because it is used as a public API with SwipeRefreshLayout.